README.md

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
In general, I'd prefer to replace `repo` with `repository`; I'd also prefer to use a less informal approach (e.g. instead of "you can build the repo", "the repo can be built")